pyproject.toml

```diff
@@ -96,6 +97,9 @@ module = "poligrain.*"
 disallow_untyped_defs = true
 disallow_incomplete_defs = true
 
+[[tool.mypy.overrides]]
+module = "numpy,numpy.testing,xarray,matplotlib,matplotlib.pyplot,matplotlib.collections"
```

Comment:
if we really want to work with type hints, it would be really good if we can get rid of this. but after quite some searching and trying things out, I still do not know how...